spec/data-model/README.md

```diff
@@ -221,53 +249,49 @@ and does not include the starting `sigil`.
 > this data model.
 
 When parsing the syntax of a _message_ that includes a _private-use annotation_
-supported by the implementation,
+or _reserved annotation_ supported by the implementation,
 the implementation SHOULD represent it in the data model
 using an interface appropriate for the semantics and meaning
 that the implementation attaches to that _annotation_.
 
+## Identifiers
+
+An `Identifier` is used to identify functions, options, attributes, markup,
+and supporting namespaces to avoid conflict between various proprietary extensions. \
```

Comment:
```suggestion
An `Identifier` is used to identify functions, options, attributes, and markup.
It also includes a namespace, to avoid conflict between various proprietary extensions.
```